crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -413,20 +504,26 @@ where
         );
     }
 
-    let channel_value = expected_outbound_amount_msat / 1000 * LIQUIDITY_MULTIPLIER;
-    ensure!(
-        channel_value <= max_channel_size_sats,
-        "Failed to open channel because maximum channel value exceeded"
-    );
+    let expected_outbound_amount =
+        Decimal::from(expected_outbound_amount_msat) / Decimal::from(1000);
+    let trade_up_to = Decimal::from(liquidity_request.trade_up_to);
+    let coordinator_leverage =
+        Decimal::try_from(liquidity_request.coordinator_leverage).expect("to fit into decimal");
```

Comment:
Not sure if it makes much difference, but I believe you could use `from_f32` instead of `try_from`.